src/holodeck/agents.py

```diff
@@ -673,11 +674,12 @@ class AgentDefinition:
         "TurtleAgent": TurtleAgent
     }
 
-    def __init__(self, agent_name, agent_type, sensors=None, starting_loc=(0, 0, 0),
+    def __init__(self, agent_name, agent_type, max_height, sensors=None, starting_loc=(0, 0, 0),
```

Comment:
By default, we want the max value feature to be disabled. There are a couple ways to do this, but the way I was thinking is to make the default max height value extremely large (see `sys.maxsize`) by default

Reply:
Tiny detail here. We'd be limiting the max height below the highest value that the engine can represent in a command. It looks to me like `sys.maxsize` should pretty much always match the max value on any given platform for `long`, but the engine takes command values in 32-bit `float`s. Since `std::numeric_limits<float>::max() > std::numeric_limits<double>::max()`, we'd be capping well below the maximum value we can handle. And because Python stores all floating-point numbers with 8 bytes, `sys.float_info.max` will give us (approximately) the max value of a `double` in C/C++. If we were really set on using a floating-point maximum, we could use `np.finfo(np.float32).max` and compare using 
```cpp
std::abs(std::numeric_limits<float>::max() - MaxHeight) < std::numeric_limits<float>::epsilon());
```
in the engine.

That being said, I'd still prefer using `sys.maxsize` as the "off" limit because it seems to me like it's good enough to tell users to keep their max height below ~9.2 quintillion even if the max values don't line up. But maybe we need a comment on the engine explaining why we're doing this unless you guys think the connection between `long` on the engine and `sys.maxsize` on the client is clear enough by itself.

Reply:
Added this in environments.py